pandas/core/indexes/base.py

```diff
@@ -4802,23 +4891,26 @@ def _add_numeric_methods_disabled(cls):
         cls.__inv__ = make_invalid_op('__inv__')
 
     def _maybe_update_attributes(self, attrs):
-        """ Update Index attributes (e.g. freq) depending on op """
+        """
+        Update Index attributes (e.g. freq) depending on op.
+        """
         return attrs
 
     def _validate_for_numeric_unaryop(self, op, opstr):
-        """ validate if we can perform a numeric unary operation """
-
+        """
+        Validate if we can perform a numeric unary operation.
+        """
         if not self._is_numeric_dtype:
             raise TypeError("cannot evaluate a numeric op "
                             "{opstr} for type: {typ}"
                             .format(opstr=opstr, typ=type(self).__name__))
 
     def _validate_for_numeric_binop(self, other, op):
         """
-        return valid other, evaluate or raise TypeError
-        if we are not of the appropriate type
+        Return valid other; evaluate or raise TypeError if we are not of
+        the appropriate type.
 
-        internal method called by ops
+        *this is an internal method called by ops*
```

Comment:
Can you add this in a `Notes` section instead of the stars? I guess saying the it's call by ops is useful in this case.